Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!groovy
```

Comment:
This file is unnecessary. I use travis CI to build and test the project - please take a look at .travis file.